responsibleai/responsibleai/managers/error_analysis_manager.py

```diff
@@ -82,6 +82,35 @@ def as_error_config(json_dict):
         return json_dict
 
 
+class MetadataRemovalModelWrapper():
```

Comment:
Will this go into `mlwrappers` eventually? @imatiach-msft
Note: this is not a blocker for this PR regardless of the answer.